src/components/PastTours/TourStep.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
```

Comment:
Wow.. insane how much more readable this makes it. Definitely was getting very redundant in pastTours. This is such a smart way to clean it up.